examples/Pipfile.valid.with-categories

```diff
@@ -0,0 +1,5 @@
+[packages]
+plette = { path = '.', extras = ['validation'], editable = true }
+
+[documentaions]
```

Comment:
Did you mean this to be `documentations`?

Reply:
Fixed. Thanks.
I also, added a change to validate package categories.